src/sentry/static/sentry/app/components/activity/note/input.jsx

```diff
@@ -280,9 +273,29 @@ class NoteInput extends React.Component {
   }
 }
 
-export {NoteInput};
+class NoteInputContainer extends React.Component {
+  static propTypes = {
+    projectSlugs: PropTypes.arrayOf(PropTypes.string),
+  };
+
+  renderInput = ({members, teams}) => {
+    const {projectSlugs: _, ...props} = this.props;
+    return <NoteInput memberList={members} teams={teams} {...props} />;
+  };
+
+  render() {
+    const {projectSlugs} = this.props;
+    const me = ConfigStore.get('user');
+
+    return (
+      <Mentionables me={me} projectSlugs={projectSlugs}>
+        {this.renderInput}
+      </Mentionables>
+    );
+  }
+}
 
-export default withApi(NoteInput);
+export default withApi(NoteInputContainer);
```

Comment:
Do you need `withApi()` on the container? It doesn't forward the prop or use it.